test/tests/server_connectorTest.js

```diff
@@ -285,4 +285,29 @@ describe("Connector Server", function () {
 			);
 		});
 	});
+	
+	describe('/connector/getTranslatorCode', function() {
+		it('should respond with translator code', function* () {
+			var code = 'function detectWeb() {}\nfunction doImport() {}';
+			var translator = buildDummyTranslator(4, code);
+			sinon.stub(Zotero.Translators, 'get').returns(translator);
```

Comment:
We should test that `Zotero.Translators.get` is getting a request for the right translator.